### PR TITLE
Describe how to Configure New Order Email Notification

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request describes how to Configure New Order Email Notification

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

https://docs.magento.com/user-guide/sales/order-status-notification.html

whatsnew
Updated the [Order Status Notification](https://docs.magento.com/user-guide/sales/order-status-notification.html) topic to include configuration of email templates for new order notifications.

### CODE REVIEW

File: src/sales/order-status-notification.md [hunk 1]
```diff
@@ -21,3 +21,27 @@ _Customer Order Status Notification_
 1. Set **Customer Order Status Notification** to `Enable`.
 
 1. When complete, click **Save Config**.
+
+## Configure New Order Email Notification
+
+1. On the Admin sidebar, go to **Stores** > _Settings_ > **Configuration**.
+
+1. In the left panel, expand **Sales** and choose **Sales Email** underneath.
```
Comment (BarnyShergold) [inline src/sales/order-status-notification.md:29]:
```suggestion
1. In the left panel, expand **Sales** and choose **Sales Emails** underneath.
```

File: src/sales/order-status-notification.md [hunk 2]
```diff
@@ -21,3 +21,27 @@ _Customer Order Status Notification_
...
+
+1. Expand ![Expansion selector]({% link images/images/btn-expand.png %}) the **Order** section and enable the order email notification.
+
+1. Set **New Order Confirmation Email Sender** to one of the following:
+
+    - General Contact
+    - Sales Representative
+    - Customer Support
```
Comment (BarnyShergold) [inline src/sales/order-status-notification.md:37]: Add -
Custom Email 1
Custom Email 2

File: src/sales/order-status-notification.md [hunk 3]
```diff
@@ -21,3 +21,27 @@ _Customer Order Status Notification_
...
+
+1. Choose the **New Order Confirmation Template** for `Customers` and `Guests`.
```
Comment (BarnyShergold) [inline src/sales/order-status-notification.md:39]: List the three options in full -
New Order (Default)
New Pickup Order 
New Pickup Order For Guest

File: src/sales/order-status-notification.md [hunk 4]
```diff
@@ -21,3 +21,27 @@ _Customer Order Status Notification_
...
+
+1. To notify about the new order, enter the email address to **Send Order Email Copy To**.
```
Comment (BarnyShergold) [inline src/sales/order-status-notification.md:41]:
```suggestion
1. To notify another person (e.g. Admin) about the new order, enter the email address to **Send Order Email Copy To**. It is possible to add multiple email addresses here if more than one additional recipient is required.
```

File: src/sales/order-status-notification.md [hunk 5]
```diff
@@ -21,3 +21,27 @@ _Customer Order Status Notification_
...
+
+1. Set the **Send Order Email Copy Method** to one of the following:
+
+    |--- |--- |
+    |Bcc|Only one email about the new order will be sent to both customer and the recipient, but the customer won't see that the email they received was also sent to the recipient.|
```
Comment (BarnyShergold) [inline src/sales/order-status-notification.md:46]:
```suggestion
    |Bcc|Only one email about the new order will be sent to both customer and the additional recipient, but the customer won't see that the email they received was also sent to the additional recipient.|
```

File: src/sales/order-status-notification.md [hunk 6]
```diff
@@ -21,3 +21,27 @@ _Customer Order Status Notification_
...
+    |Separate Email|Two separate emails will be sent to the recipient and the customer.|
```
Comment (BarnyShergold) [inline src/sales/order-status-notification.md:47]:
```suggestion
    |Separate Email|Two separate emails will be sent, one to the recipient and one to the customer.|
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
A few suggested changes.

Please note - if a suggested change is accepted then it is better to use the "accept" option on the PR so it's easy to see - if you add the changes manually and then push, then they have to be rechecked as it's possible a small mistake could be made.

Thanks!

**Comment by ZhovnirMaria** [comment]
Done ✅ Thanks!

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Pull request merged**